src/ansible_dev_environment/cli.py

```diff
@@ -109,21 +109,26 @@ def ensure_isolated(self) -> None:
             self.output.hint(hint)
             errored = True
 
-        home_coll = Path.home() / ".ansible/collections/ansible_collections"
-        if home_coll.exists() and tuple(home_coll.iterdir()):
-            err = f"Collections found in {home_coll}"
-            self.output.error(err)
-            hint = "Run `rm -rf ~/.ansible/collections` to remove them."
-            self.output.hint(hint)
-            errored = True
+        ansible_home: Path
+        if "ANSIBLE_HOME" not in os.environ:
+            ansible_home = Path(os.environ.get("VIRTUAL_ENV", "~/.ansible"))
+            self.output.warning(
+                f"Declaring ANSIBLE_HOME={ansible_home} to ensure isolation from user environment.",
+            )
+            os.environ["ANSIBLE_HOME"] = str(ansible_home)
```

Comment:
this would only persis during the run of ADE but not after,  can we leave something in the cwd to isolate it for other processes?

Reply:
like an ansible.cfg file that sets the ANSIBLE_HOME and nullifies the COLLECTION_PATHS?